exercises/practice/triangle/.meta/example.cairo

```diff
@@ -0,0 +1,41 @@
+pub fn is_equilateral(sides: Array<u64>) -> bool {
```

Comment:
```suggestion
pub fn is_equilateral(sides: [u64, 3]) -> bool {
```
Since we are working with triangles, i.e. 3-item arrays, we can make `sides` a fixed-size array. This applies to all